setup.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+echo "Setting everything up to run Tournesol:"
+echo "Step 1: Install the appropriate version of Mozilla Firefox."
+echo "Step 2: Install the apt dependencies."
+echo "Step 3: Install nodejs dependencies for the frontend."
+echo "Step 4: Create a new python3 venv and install the python dependencies for the backend."
+
+echo " "
+echo "=================================================="
+echo "Step 1 / 4: Install the appropriate version of Mozilla Firefox."
+echo " "
+# Install the appropriate version of Firefox
+if [ "$(firefox --version)" != "Mozilla Firefox 84.0.1" ]; then
+    echo "Installing Mozilla Firefox version 84.0.1"
+	wget -c https://download-installer.cdn.mozilla.net/pub/firefox/releases/84.0.1/linux-x86_64/en-US/firefox-84.0.1.tar.bz2
+    tar -xvf firefox-84.0.1.tar.bz2
+    sudo ln -fs $(pwd)/firefox/firefox /usr/bin/firefox
+else
+    echo "OK: Mozilla Firefox version 84.0.1 already installed."
+fi
```

Comment:
I don't think we want to force contributors to install firefox. Also some of the contributors (me) are not using linux machines

Reply:
Firefox is there for the integration tests to run

Reply:
Yes, but we can't block mac users at the moment, so I recommend this should be put bac in the CI script __ for now __

Reply:
Maybe we just install it locally instead and add to PATH

Reply:
Since we will use Docker in the future, it would not matter, so it's fine to install it globally here